aries-site/src/themes/aries.js

```diff
@@ -6,10 +6,42 @@ export const aries = deepMerge(hpe, {
   // To be stripped out once theme changes are made in grommet-theme-hpe
   // keeping file for use as playground for future theme adjusments that need
   // to be quickly tested
-  menu: {
-    group: {
-      separator: {
-        color: 'border-weak',
+  feedback: {
+    body: {
+      gap: 'medium',
+    },
+    button: {
```

Comment:
```suggestion
    closeButton: {
```
should we be more descriptive here since there are other buttons in the modal?